server/src/Assets/entities/asset.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { User } from '../../users/entities/user.entity';
+
+@Entity('assets')
+export class Asset {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column('text')
+  title: string;
+
+  @Column('text')
+  descripton: string;
+
+  // @Column('text')
+  // type: string;
+
+  // @Column('text')
+  // image_url: string;
+
+  // @Column('text')
+  // address: string;
+
+  @ManyToOne(() => User, (user) => user.assets)
+  poster_id: User;
```

Comment:
looks like somewhere along the line (maybe typeorm 🤷 ) the `id` suffix gets added for us, so let's have this just be `poster`
<img width="845" alt="Screen Shot 2021-10-20 at 6 19 25 PM" src="https://user-images.githubusercontent.com/16400333/138194474-2260c07d-940a-4f28-9a17-d0b95b1e2e0c.png">